### PR TITLE
mon/MgrMonitor: add 'mgr dump [epoch]' command

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: src/mon/MgrMonitor.cc [hunk 1]
```diff
@@ -444,8 +444,61 @@ void MgrMonitor::drop_standby(uint64_t gid)
...
+  MonSession *session = m->get_session();
+  if (!session) {
+    mon->reply_command(op, -EACCES, "access denied", rdata,
+		       get_last_committed());
+    return true;
+  }
 
+  string format;
+  cmd_getval(g_ceph_context, cmdmap, "format", format, string("json-pretty"));
+  derr << "format " << format << dendl;
```
Comment (tchaikov) [inline src/mon/MgrMonitor.cc]: i believe this is only for debugging.

**Comment by liewegas** [comment]
oops fixed

**Comment by tchaikov** [comment]
could have a simple test in `qa/workunits/cephtool/test.sh`

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2017-05-18_23:49:52-rados-wip-yuri-testing_2017_5_19-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2017-05-19_03:47:57-rados-wip-yuri-testing_2017_5_19---basic-smithi/

**Pull request merged**